aries-site/src/pages/components/selectmultiple.mdx

```diff
@@ -0,0 +1,47 @@
+import { Example } from '../../layouts';
+import { SelectMultipleAnatomyExample, SelectMultipleSimpleExample } from '../../examples';
+
+The SelectMultiple component is typically used as a form element but can also be leveraged for user experiences that include searching or narrowing down data. It contains an optional search field. When within a form, the SelectMultiple will support labels, error and disabled states.
+
+<Example
+  componentName="SelectMultiple"
+  code="https://raw.githubusercontent.com/grommet/hpe-design-system/master/aries-site/src/examples/components/selectmultiple/SelectMultipleSimpleExample.js"
+  docs="https://v2.grommet.io/selectmultiple?theme=hpe#props"
+  grommetSource="https://github.com/grommet/grommet/blob/master/src/js/components/SelectMultiple/SelectMultiple.js"
+>
+  <SelectMultipleSimpleExample />
+</Example>
+
+## When to use
+
+When multiple selections can be made in a list that contains 5 or more items.
+
+## When not to use
+
+- For lists that contain less than 5 options to choose from, consider using a [CheckBoxGroup](/components/checkboxgroup) instead. 
```

Comment:
is there a case where I would use the regular select? but with `multiple` prop? 
or are we not recommending that at all?

Reply:
I'll have to let someone else comment on that. The guidance is just from the Figma for this page.

Reply:
I believe for we are encouraging the SelectMultiple component to be used instead of Select with the `multiple` prop